public/pages/intro-class.html

```diff
@@ -211,13 +212,13 @@ <h4>Refcode Bootcamp</h4>
           <p class="timeframe">(<i>3 months</i>)</p>
           <p>Graduates are eligible to apply for the Refcode Bootcamp, where you will learn additional coding skills. You can also apply to be a Refcode Bootcamp Resident. Residents are paid a stipend of $1,500 per month.</p>
 
-          <h4>Refcode Software Agency Technician</h4>
+          <h4>Refcode Software Agency  Technician</h4>
```

Comment:
Adding this space is not necessary. It would be good to look over your own "diff" and review your own code to find these little things and eliminate them.